molecule/default/playbook.yml

```diff
@@ -15,3 +15,8 @@
 
   roles:
     - role: elasticsearch
+      sansible_elasticsearch_additional_config:
+        bootstrap.mlockall: "true"
```

Comment:
Can we test this with `yes` instead of `"true"`, please?

Reply:
Ah yes I forgot about that, this is going to be hard due to the way we handle the config, I wonder if to_nice_yaml gets around this issue?